pages/_document.tsx

```diff
@@ -21,10 +21,14 @@ function Document() {
           defer
         ></script>
         {
-        <script
-          id="ze-snippet"
-          src="https://static.zdassets.com/ekr/snippet.js?key=3ca3ab96-67f1-4a75-a4ab-8e206a0a4660"
-          async> </script>}
+          <script
+            id="ze-snippet"
+            src="https://static.zdassets.com/ekr/snippet.js?key=3ca3ab96-67f1-4a75-a4ab-8e206a0a4660"
+            async
+          >
+            {' '}
```

Comment:
@chamibia what's this change for?

Reply:
i just cloned the repo locally now and updated the next file. I think it might be from prettier when i ran yarn lint as I didnt change any code in that file , just in the next config. It needs to be reverted?

Reply:
yeah, that has nothing to do there